### DISCUSSION TITLE
Feature request: Change browser opened based on current Space

### DISCUSSION THREAD

**Discussion by JohnLockeNJ** [will-stone/browserosaurus#488] [Ideas]
Could you add a feature that changes the browser opened based on what Space you are in?

I have different Spaces/virtual desktops (ie Desktop 1, Desktop 2, etc. in Mission Control) for Personal, Job 1, and Job 2, and I run different browsers in each. It would be great if links I click on from any program in a given Space only opened the browser that I would assign to that Space in browserosaurus instead of picking each time.

**Comment by will-stone** [comment]
hi @JohnLockeNJ Thanks for using Browserosaurus. I like to keep the consistency that the picker will always show, and adding rules is out of scope for this project: https://github.com/will-stone/browserosaurus/discussions/476
